### DISCUSSION TITLE
Terribly buggy interface in Linux

### DISCUSSION THREAD

**Discussion by Leonardo5681** [86Box/86Box#4463] [Q&A] [answered]
### What happened?

Yesterday I wanted to try 86Box, I opened it and the ROM download banner had an interface that didn't enter and you couldn't press anything. I downloaded the ROMs and it opens, but the interface is unusable, with unreadable texts and penetrated buttons.
![image](https://github.com/86Box/86Box/assets/92219720/838e7901-6b6e-4f94-b21a-060f6754d3ae)
![image](https://github.com/86Box/86Box/assets/92219720/2e42cca4-d432-4a7a-b7ff-f01e9422675e)
![image](https://github.com/86Box/86Box/assets/92219720/5fb7d84e-cb34-475b-aba7-40e2e9004c58)
![image](https://github.com/86Box/86Box/assets/92219720/d4713252-675e-4816-b1e3-615e00588f31)
![image](https://github.com/86Box/86Box/assets/92219720/1930f8e3-e770-4dcb-b6c2-3e2b86f0a68c)
![image](https://github.com/86Box/86Box/assets/92219720/aec070ee-0186-4e19-953d-da20e02a8d69)

### Configuration file

```ini
[General]
vid_renderer = qt_software

[Machine]
machine = ibmpc
cpu_family = 8088
cpu_speed = 4772728
cpu_multi = 1
cpu_use_dynarec = 0
fpu_softfloat = 0
time_sync = local
mem_size = 64

[Video]
gfxcard = cga

[Input devices]
mouse_type = none

[Sound]
fm_driver = nuked

[Network]
net_01_link = 0
net_02_link = 0
net_03_link = 0
net_04_link = 0

[Storage controllers]
hdc = none
cassette_enabled = 1
cassette_mode = load
```

### Operating system

Fedora Linux 40 Workstation Edition

### CPU

AMD Ryzen 5 5500U

### 86Box version

v4.1.1 5634

### Build architecture

Linux - x64 (64-bit)

### Build type

- [X] New recompiler
- [ ] Debug build

### Download source

Official website (Jenkins, GitHub)

### Additional context

Obviously I installed Qt, do I need any other dependencies?

**Comment by OBattler** [comment]
I have no idea. We have many Linux users and you're the first one I see reporting this kind of problem.

**Comment by OBattler** [comment]
You most likely did not properly set up Qt theming.

**Comment by cold-brewed** [comment]
How are you running 86Box? Is it with the official AppImage?

I can't reproduce this on Fedora 39 with plasma and the AppImage. Is there perhaps some type of fractional scaling in place that's confusing Qt?

You could also try to force Qt to x11 by setting the environment variable `QT_QPA_PLATFORM="xcb"`, so something like:

```
QT_QPA_PLATFORM="xcb" /path/to/AppImage <args>
```

**Comment by OBattler** [comment]
Converting this to a discussion since itº s not a bug on 86Box's side.

**Comment by aubade** [comment]
I'm having this same issue with the official appimage (i've tried the last 3 releases), on Ubuntu-Mate 22.04. I've tried the QT_QPA_PLATFORM="xcb" trick but it gives the same results.

EDIT: I really only need the UI for configuring the machine. if the cfg file options were documented somewhere I'd be fine just editing my 86box.cfg.

**Comment by Sunspark-007** [comment, marked as answer]
It's a scaling issue and your issue is complicated by the use of GTK as your desktop environment.

Try using "export QT_ENABLE_HIGHDPI_SCALING=0", see if that works for you.

There are other scaling variables available that can be adjusted. Windows has scaling issues too in fullscreen which is fixable with a different QT scaling option.

**Reply by aubade** [reply]
That fixed it, thank you.
